src/jit/importer.cpp

```diff
@@ -13594,7 +13594,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         op1->gtFlags |= GTF_ORDER_SIDEEFF; // Prevent this from being reordered
                         op1->gtFlags |= GTF_IND_VOLATILE;
                     }
-                    if (prefixFlags & PREFIX_UNALIGNED)
+                    if ((prefixFlags & PREFIX_UNALIGNED) && !varTypeIsByte(lclTyp))
```

Comment:
This change is fine, as we just want to set things up so that we don't set `GTF_IND_UNALIGNED `when we have a byte sized indirection